Makefile

```diff
@@ -204,6 +216,13 @@ install-tools:
 	cd $(TOOLS_MOD_DIR) && GOBIN=$(TOOLS_BIN_DIR) go install honnef.co/go/tools/cmd/staticcheck@v0.2.0
 	cd $(TOOLS_MOD_DIR) && GOBIN=$(TOOLS_BIN_DIR) go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.0
 	cd $(TOOLS_MOD_DIR) && GOBIN=$(TOOLS_BIN_DIR) go install mvdan.cc/sh/v3/cmd/shfmt@latest
+	cd $(TOOLS_MOD_DIR) && GOBIN=$(TOOLS_BIN_DIR) go install go.opentelemetry.io/build-tools/dbotconf
+
+.PHONY: install-dbotconf
+install-dbotconf:
```

Comment:
Is this needed?  Shouldn't it be covered by `install-tools`?

Reply:
I think I prefer the approach taken by the [opentelemetry-go repo](https://github.com/open-telemetry/opentelemetry-go/blob/main/Makefile#L31-L72).  This builds the specific version of the tool required in an ignored directory directly in the repo, which also allows for only building the tools that are needed.  I think it has the same issue with not detecting updates, but I think that can be fixed by depending on `internal/tools/go.sum`.

Reply:
I can see how this makes more sense and think we should adapt this into our process. Do you believe this is a blocker for this PR? We could open a new issue to track this improvement for a separate PR.

Reply:
Let's tackle that on a separate PR.